client/ts/graph_chart.ts

```diff
@@ -0,0 +1,64 @@
+import * as d3 from "d3";
+import {Color} from "./styles";
+
+export function generateGraghChart(jsonUrl){
+
+    // let svg = d3.select('svg'),
+    //     width = document.getElementById("drawing-section").clientWidth,
+    //     height = document.getElementById("drawing-section").clientHeight,
+    //     nodeRadius = 10;
+    // svg.attr("width", width).attr("height", height);
+    let svg = d3.select('svg'),
+        height = +svg.attr("height"),
```

Comment:
`= +`?

Reply:
to make sure the height is positive value

Reply:
Neat!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Unary_plus_2